content-src/components/DiscoveryStreamComponents/CardGrid/CardGrid.jsx

```diff
@@ -26,9 +26,20 @@ export class CardGrid extends React.PureComponent {
         source={rec.domain} />
     ));
 
+    let divisibility = ``;
+
+    if (this.props.items % 4 === 0) {
```

Comment:
Out of curiosity, is it possible to have a 12-column layout with either of these sizes or should it always go with 4?